package.json

```
@@ -26,7 +26,7 @@
     "optimize:svg": "find assets -type f -name '*.svg' -print0 | xargs -0 -n 1 -P 6 svgo --multipass --pretty --enable=removeTitle",
```

Windows does not allow single quotes in runtime arguments, so in order to work cross-platform these have to be "
```suggestion
    "optimize:svg": "find assets -type f -name \"*.svg\" -print0 | xargs -0 -n 1 -P 6 svgo --multipass --pretty --enable=removeTitle",
```